lms/static/scripts/frontend_apps/api-types.js

```diff
@@ -55,9 +55,15 @@
  * @prop {string} title
  * @prop {string} [subtitle]
  *
+ * @typedef JSTORRelatedItemsInfo
```

Comment:
The typing in this module corresponds to what we expect the response from the proxy metadata API to look like.